lib/node_modules/@stdlib/math/stats/anova1/lib/print.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+// MODULES //
+
+var isPositiveInteger = require( '@stdlib/assert/is-positive-integer' );
+var hasOwnProp = require( '@stdlib/assert/has-own-property' );
+var roundn = require( '@stdlib/math/base/special/roundn' );
+var isBoolean = require( '@stdlib/assert/is-boolean' );
+
+
+// MAIN //
+
+/**
+* Returns a function to pretty print test results.
+*
+* @private
+* @param {Object} results - test results
+* @returns {Function} pretty print function
+*/
+function prettyPrint( results ) {
+	return print;
+
+	/**
+	* Pretty-print output of ANOVA.
+	*
+	* @private
+	* @param {Object} [opts] - options object
+	* @param {PositiveInteger} [opts.numdigits=4] - number of digits after the decimal point
+	* @param {boolean} [opts.decision=true] - Whether or not to print the decision
+	* @returns {string} formatted results
+	*/
+	function print( opts ) {
+		var decision;
+		var dgts;
+		var str;
+
+		// Was unable to use the isObject method as it crashes on my installation. More information in the TODO
```

Comment:
Please remove this comment (not sure what this related to).

Reply:
Resolved.